sql-cli/CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 # Changelog
 
+## 0.2.0
+
+A feature release containing the following major features:
+
+* load files to DB without Python
+* dynamic DAG generation
+
+### Features
+
+* Integrate load file operator together with generate and run in SQL CLI
```

Comment:
Should we also link relevant PRs for the logs here? I see we do them in other Change logs.

Reply:
I would rather automate it at some point when we also automate the release process.